control/statefbk.py

```diff
@@ -589,9 +589,9 @@ def create_statefbk_iosystem(
     This function creates an input/output system that implements a
     state feedback controller of the form
 
-        u = ud - K_p (x - xd) - K_i integral(C x - C x_d)
+    .. math :: u = u_d - K_p (x - x_d) - K_i \int(C x - C x_d)
```

Comment:
This is probably OK, but see the general comment regarding keeping equations to a minimum.